<?xml version="1.0"?>

<launch>

    <!-- Navstack -->
    <node pkg="move_base" type="move_base" respawn="false" name="move_base">
        <!-- local planner -->
        <!-- costmap layers -->    
        <rosparam file="$(find tianracer_navigation)/param/local_costmap_params.yaml" command="load"/>
        <rosparam file="$(find tianracer_navigation)/param/global_costmap_params.yaml" command="load"/> 
        <!-- move_base params -->
        <rosparam file="$(find tianracer_navigation)/param/move_base_params.yaml" command="load"/>
    </node>

    <!-- L1 controller -->  
    <node pkg="tianracer_navigation" type="L1_controller_v2" respawn="false" name="L1_controller_v2" output="screen">       
        <!-- L1 -->
        <param name="Vcmd" value="1.0" /> <!-- speed of car m/s -->        
        <!-- ESC -->
	<param name="base_speed" value="1.0"/> <!-- speed of car m/s  --> 
        <!-- Servo -->
        <param name="base_angle" value="0.0"/> <!-- the middle pos of servo if tuning needed  -->
        <param name="angle_gain" value="-3.5"/> <!-- for tt02: >0, for hsp: <0 -->		

        <!-- remap from="/odometry/filtered" to="odom" / --> 
        <remap from="/move_base_node/NavfnROS/plan" to="/move_base/NavfnROS/plan" /> 
    </node>

</launch>
